### PR TITLE
feat: add a spring security quickstart

### PR DESCRIPTION
**Pull request by aureamunoz**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the documentation must not be updated
- [x] links the documentation update pull request (if needed)
- [x] updates or creates the `README.md` file (with build and run instructions)
- [x] For new quickstart, is located in the directory _component-quickstart_
- [x] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by aureamunoz** [comment]
The PR of the `spring-security` extension guide: https://github.com/quarkusio/quarkus/pull/5792

**Comment by aureamunoz** [comment]
@geoand @gsmet :point_up_2:

**Comment by cescoffier** [comment]
CC @danielpetisme

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ See [CONTRIBUTING](CONTRIBUTING.md) for how to build these examples:
 * [Spring DI compatibility layer](./spring-di-quickstart): How to use our Spring Dependency Injection compatibility layer
 * [Spring Data extension](./spring-data-jpa-quickstart): How to the Quarkus extension for the Spring Data API
 * [Spring Web extension](./spring-web-quickstart): How to the Quarkus extension for the Spring Web API
+* [Spring Security extension](./spring-security-quickstart): How to the Quarkus extension for the Spring Security API
```
Comment (gsmet) [inline README.md]: Could you add `use` after `How to` for the 3 Spring quickstarts?

File: spring-security-quickstart/pom.xml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      <artifactId>quarkus-spring-security</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-spring-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-elytron-security-properties-file</artifactId>
+    </dependency>
```
Comment (gsmet) [inline spring-security-quickstart/pom.xml]: Let's get all the runtime dependencies before the test ones?

**Comment by geoand** [comment]
Nope, I don't have anything more to add

**Comment by gsmet** [comment]
I force pushed an update. That will be one less item on my list.

**Comment by aureamunoz** [comment]
Thank you!

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**